src/models/converters/transactions/mod.rs

```diff
@@ -45,72 +45,35 @@ impl MultisigTransaction {
     }
 
     fn transaction_info(&self, info_provider: &mut dyn InfoProvider) -> TransactionInfo {
-        if self.is_settings_change() {
-            // Early exit if it is a setting change
-            return TransactionInfo::SettingsChange(self.to_settings_change());
-        }
-        let token = info_provider.token_info(&self.to).ok();
```

Comment:
It seems like we were always trying to fetch the `token_info` regardless whether it was available in the `MultisigTransaction`. Should we keep this behaviour? Or only fetch when there is missing info? @rmeissner

Reply:
I would always load it .... it is simpler for now